### ISSUE TITLE
Table session column data

### ISSUE THREAD

**Issue by pekka2** [opencart/opencart#7116]
Tbale 'session', line 5260

https://github.com/opencart/opencart/blob/master/upload/system/helper/db_schema.php#L5260

column data:
`'type' => 'text',`
change to:
`'type' => 'longtext',`

**Comment by MrPlow254** [comment]
Why would you pick longtext? 

https://dev.mysql.com/doc/refman/5.7/en/string-type-overview.html

```
      Type | Maximum length
-----------+-------------------------------------
  TINYTEXT |           255 (2 8−1) bytes
      TEXT |        65,535 (216−1) bytes = 64 KB
MEDIUMTEXT |    16,777,215 (224−1) bytes = 16 MB
  LONGTEXT | 4,294,967,295 (232−1) bytes =  4 GB
```

Does a session data field really go above 16MB? A longtext uses an extra byte per row every regardless of how much is uses vs a mediumtext

**Comment by danielkerr** [comment]
yep some one else posting without checking or stating why! going to remove.
